### PR TITLE
Added auto option in schema.yaml

### PR DESCRIPTION
**Pull request by govarsha**
Added auto option in schema.yaml and added a function in utils.py to select arima if number of cols > 15, otherwise automlx

### CODE REVIEW

**Review by ahosler — APPROVED** [review]
Looks awesome! We will continue to update the auto-select method as we go.

**Review by prasankh — APPROVED** [review]

**Review by codeloop — APPROVED** [review]

File: ads/opctl/operator/lowcode/forecast/utils.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 )
 
 from ads.dataset.label_encoder import DataFrameLabelEncoder
+from ads.opctl.operator.lowcode.forecast.const import SupportedModels
```
Comment (mrDzurb) [inline ads/opctl/operator/lowcode/forecast/utils.py:24]: I think here would be better to use relative imports. Something like this.
```
from .const import SupportedModels
```

File: ads/opctl/operator/lowcode/forecast/utils.py [hunk 2]
```diff
@@ -291,3 +292,8 @@ def human_time_friendly(seconds):
             )
     accumulator.append("{} secs".format(round(seconds, 2)))
     return ", ".join(accumulator)
+
+def select_auto_model(columns):
```
Comment (mrDzurb) [inline ads/opctl/operator/lowcode/forecast/utils.py:296]: Could you add some description for this method? Also I would suggest to move ```15``` into constants.

File: ads/opctl/operator/lowcode/forecast/utils.py (hunk 1 above)
Comment (govarsha) [reply ads/opctl/operator/lowcode/forecast/utils.py:24]: @mrDzurb Yeah I will change that.

File: ads/opctl/operator/lowcode/forecast/utils.py (hunk 2 above)
Comment (govarsha) [reply ads/opctl/operator/lowcode/forecast/utils.py:296]: @mrDzurb I will move 15 into constants. Where should I add description? Should I just add a comment there?

**Pull request merged**